tests/Integration/Behaviour/Features/Context/Domain/ThemeMailTemplatesFeatureContext.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Tests\Integration\Behaviour\Features\Context\Domain;
+
+use Behat\Gherkin\Node\TableNode;
+use PHPUnit_Framework_Assert;
+use PHPUnit_Framework_AssertionFailedError;
+use PrestaShop\PrestaShop\Core\Domain\MailTemplate\Command\GenerateThemeMailTemplatesCommand;
+use RuntimeException;
+use Symfony\Component\DependencyInjection\Exception\ServiceCircularReferenceException;
+use Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException;
+
+class ThemeMailTemplatesFeatureContext extends AbstractDomainFeatureContext
+{
+    /**
+     * @When I generate emails with the following properties:
+     *
+     * @param TableNode $table
+     *
+     * @throws RuntimeException
+     * @throws ServiceCircularReferenceException
+     * @throws ServiceNotFoundException
+     */
+    public function iGenerateEmailsWithTheFollowingProperties(TableNode $table)
+    {
+        $data = $this->extractFirstRowFromProperties($table);
+
+        $overwriteTemplates = $data['overwrite_templates'] === '' ? false : $data['overwrite_templates'];
```

Comment:
I think it can be shortened using `?:` operator 🤔?

Reply:
Elvis operator is great here :) supported from php 5.3